app/views/estimate_flow/property_entry.html.slim

```diff
@@ -6,7 +6,7 @@
 .govuk-grid-column-two-thirds
   = render "estimate_flow/forms/property_entry",
           i18n_key: "property_entry",
-          show_smod: true,
+          show_smod: !@estimate.upper_tribunal?,
```

Comment:
are `show_smod` and `show_smod_content` different in any way? If not maybe we could take this opportunity to make that consistent